am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -74,7 +78,8 @@ public AccessManagementService(DataSource dataSource) {
      */
     @AuditLog("explicit access granted by '{{mdc:caller}}' to resource '{{accessGrant.resourceId}}' "
         + "defined as '{{accessGrant.serviceName}}|{{accessGrant.resourceType}}|{{accessGrant.resourceName}}' "
-        + "for accessors '{{accessGrant.accessorIds}}': {{accessGrant.attributePermissions}}")
+        + "for accessors '{{accessGrant.accessorIds}}' in relationship '{{accessGrant.relationship}}': "
```

Comment:
I intentionally left `permissions` as last piece in template because it could get very long.

This way it is also similar to what default permission grant audits:

```
default role access granted by '{{mdc:caller}}' to resource "
        + "defined as '{{accessGrant.serviceName}}|{{accessGrant.resourceType}}|{{accessGrant.resourceName}}' "
        + "for role '{{accessGrant.roleName}}': {{accessGrant.attributePermissions}}
```

I treat relationship for explicit grant same as role for default role based grant.

Reply:
ok, permissions is fine to remain at the end. I still have issue with the word in however. 

"For accessors {{}} and relationship"?
"For accessors {{}} because of relationship"?
"For accessors {{}} with relationship"?